### DISCUSSION TITLE
CONGRATULATIONS! django-ninja is the biggest gaining Django application in 2023 :tada:

### DISCUSSION THREAD

**Discussion by benjaoming** [vitalik/django-ninja#1141] [General]
I'm watching a live discussion [1] of the Django Developer's Survey 2023 [2], and I can't help but to share the really good reputation that django-ninja is building. It went from 0 to 6% of 4,000 respondents' favorite application.

Everyone in this community should be proud, but especially @vitalik :raised_hands: 

[1] Live video discussion: https://www.youtube.com/watch?v=2at9FtNncbc
[2] Survey: https://lp.jetbrains.com/django-developer-survey-2023/

**Comment by vitalik** [comment]
Wow - really inspiring ! Thank you for sharing
